src/scss/components/Card-mobile.scss

```diff
@@ -0,0 +1,99 @@
+@use "../abstracts/fonts.scss";
+@use "../abstracts/colors.scss";
+@import "../abstracts/breakpoints.scss";
+
+.product-card {
+  align-items: flex-start;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  height: 22.1rem;
```

Comment:
Wysokość nie powinna być na sztywno, jeśli do komponentu trafi więcej tekstu, to layout się sypnie:
<img width="412" alt="Screen Shot 2022-02-16 at 22 10 19" src="https://user-images.githubusercontent.com/7902324/154357677-a9236ebb-ad0d-49d0-9b23-48f05c45a012.png">